front-end/src/App.tsx

```diff
@@ -63,6 +64,7 @@ function App() {
     if (nodes.length === 0){
       localStorage.removeItem(LOCAL_STORAGE_NODES);
```

Comment:
TODO: `AxiosWrapper.nodes = []`, so that the wrapper loses its credentials when a logout happens.

Reply:
Done.